File: app/src/main/java/com/wineguesser/deductive/model/wine/red/RedWineDescriptors.java
package com.wineguesser.deductive.model.wine.red;

import com.wineguesser.deductive.model.wine.WineDescriptors;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.NoArgsConstructor;

@Data
@EqualsAndHashCode(callSuper = true)
@NoArgsConstructor
@AllArgsConstructor
public class RedWineDescriptors extends WineDescriptors {
    private Color color;
    private SecondaryColor secondaryColor;
    private Tannin tannin;
}

File: app/src/main/java/com/wineguesser/deductive/model/wine/red/RedWineDescriptors.kt
package com.wineguesser.deductive.model.wine.red

import com.wineguesser.deductive.model.wine.WineDescriptors

class RedWineDescriptors : WineDescriptors() {
    var color: Color? = null
    var secondaryColor: SecondaryColor? = null
    var tannin: Tannin? = null
}
